lib/logstash/filters/yaml.rb

```diff
@@ -58,42 +59,43 @@ def filter(event)
 
     return unless event.include?(@source)
 
-    source = event[@source]
+    source = event.get(@source)
+
+    begin
+      unmarshalled = YAML::load(source)
+    rescue => e
+      event.tag("_yamlparsefailure")
+      @logger.warn("Trouble parsing yaml", :source => @source,
```

Comment:
"Trouble" vs "Error"?

Reply:
Can I suggest using [GitHub change suggestions](https://twitter.com/github/status/1054794897448939520?s=19)? :D

Reply:
you may!! 😁